packages/drizzle/src/queries/parseParams.ts

```diff
@@ -138,9 +138,12 @@ export async function parseParams({
                   let formattedValue = val
                   if (adapter.name === 'sqlite' && operator === 'equals' && !isNaN(val)) {
                     formattedValue = val
-                  }
-                  if (['in', 'not_in'].includes(operator) && Array.isArray(val)) {
-                    formattedValue = `(${val.map((v) => `'${v}'`).join(', ')})`
+                  } else if (['in', 'not_in'].includes(operator) && Array.isArray(val)) {
+                    if (adapter.name === 'sqlite') {
+                      formattedValue = `(${val.map((v) => `${v}`).join(',')})`
+                    } else {
+                      formattedValue = `(${val.map((v) => `'${v}'`).join(', ')})`
```

Comment:
Wouldn't `formattedValue = `(${val.map((v) => `'${v}'`).join(', ')})`` work for both postgres and sqlite and so we can remove the if block and just use the format that works for both?

Reply:
nvm, extra `'` and ` ` makes this necessary, thanks!